Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -96,7 +96,7 @@ extension ModuleError: CustomStringConvertible {
                 (cycle.path + cycle.cycle).joined(separator: " -> ") +
                 " -> " + cycle.cycle[0]
         case .invalidPublicHeadersDirectory(let name):
-            return "public headers directory path for '\(name)' is invalid or not contained in the target"
+            return "public headers or include directory path for '\(name)' is invalid or not contained in the target"
```

Comment:
The new configuration is a little confusing, I think, since it can be interpreted as implying that the public-headers directory and the include directory are two different things.  If we want to mention the word "include" here, could we formulate it as saying `public headers ("include") directory` instead?

Reply:
makes sense, will update